packages/react-components/src/organisms/__tests__/EventCard.test.tsx

```diff
@@ -26,3 +27,91 @@ it('renders the event tags', () => {
   const { getByText } = render(<EventCard {...props} tags={['MyTag']} />);
   expect(getByText('MyTag')).toBeVisible();
 });
+
+describe('event is occurring', () => {
+  it('toasts for meetings without meeting link', () => {
+    const { getByText, getByTitle } = render(
+      <EventCard
+        {...props}
+        status="Confirmed"
+        meetingLink={undefined}
+        startDate={subMinutes(new Date(), 1).toISOString()}
+        endDate={addMinutes(new Date(), 1).toISOString()}
+      />,
+    );
+    expect(getByText(/currently happening/i)).toBeVisible();
+    expect(getByTitle('Clock')).toBeInTheDocument();
+  });
+  it('toasts for meetings with meeting link', () => {
+    const { getByText, getByTitle } = render(
+      <EventCard
+        {...props}
+        status="Confirmed"
+        meetingLink="http://example.com"
+        startDate={subMinutes(new Date(), 1).toISOString()}
+        endDate={addMinutes(new Date(), 1).toISOString()}
+      />,
+    );
+    expect(getByText(/currently happening/i)).toBeVisible();
+    expect(getByTitle('Clock')).toBeInTheDocument();
+    expect(getByText(/join/i)).toHaveAttribute('href', 'http://example.com');
+  });
+});
+
+describe('event is over', () => {
+  it('toasts meeting materials coming soon', () => {
+    const { getByText, getByTitle } = render(
+      <EventCard
+        {...props}
+        status="Confirmed"
+        meetingLink="http://example.com"
+        meetingMaterials={[]}
+        presentation={undefined}
+        notes={undefined}
+        videoRecording={undefined}
+        startDate={subDays(new Date(), 2).toISOString()}
+        endDate={subDays(new Date(), 1).toISOString()}
+      />,
+    );
+    expect(getByText(/coming soon/i)).toBeVisible();
+    expect(getByTitle('Paper Clip')).toBeInTheDocument();
+  });
+  it('toasts number of available materials', () => {
+    const { getByText, getByTitle } = render(
+      <EventCard
+        {...props}
+        status="Confirmed"
+        meetingLink="http://example.com"
+        meetingMaterials={[
+          { title: '123', url: 'http://example.com' },
+          { title: '345', url: 'http://example.com' },
+        ]}
+        presentation={'presentation'}
+        notes={undefined}
+        videoRecording={undefined}
+        startDate={subDays(new Date(), 2).toISOString()}
+        endDate={subDays(new Date(), 1).toISOString()}
+      />,
+    );
+    expect(getByText(/materials \(2\)/i)).toBeVisible();
```

Comment:
why is this two, it seems like it should be 3? (1 presentation + 2 additional)

Reply:
I did that originally. Turns out the ticket says it should be a maximum of 4 so 2 additional = 1

Reply:
It doesn't though :D